mirrord/layer/src/load.rs

```diff
@@ -1,18 +1,44 @@
 use mirrord_config::{util::VecOrSingle, LayerConfig};
 use tracing::trace;
 
+mod build_tools {
+    use std::{collections::HashSet, sync::LazyLock};
+
+    static BUILD_TOOL_PROCESSES: LazyLock<HashSet<&str>> = LazyLock::new(|| {
+        HashSet::from([
+            "cc",
+            "ld",
+            "go",
+            "air",
+            "asm",
+            "math",
+            "cargo",
+            "hpack",
+            "rustc",
+            "compile",
+            "cargo-watch",
+        ])
+    });
+
+    pub fn is_dev_tool(given_process: &str) -> bool {
```

Comment:
Why don't you just use BUILD_TOOL_PROCESSES.contains directly? Also why is it called is_dev_tool and not is_build_tool?

Reply:
The only one I didn't change I figured dev-tools is a bit incorrect